README.md

```diff
@@ -39,6 +39,7 @@ A curated list of awesome online implementations of integrated development envir
 * [Datalore](https://datalore.io/) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run Python notebooks for data analysis and visualization, with real-time collaboration, incremental calculation, and integrated version control
 * [MATLAB Online](https://www.mathworks.com/products/matlab-online.html) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run MATLAB code, visualize results, share scripts and collaborate on mathematical computing software
 * [Colaboratory](https://research.google.com/colaboratory/) [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/googlecolab/colabtools) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run Python in a Jupyter notebook with ML libraries and free GPUs for running experiments
+* [Superblocks Studio](https://superblocks.com/studio) [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/SuperblocksHQ/studio) - Superblocks studio is an open source online IDE developed for Ethreum DApp developers. Studio helps you code, build, and deploy your smart contracts for the Ethereum blockchain
```

Comment:
The description should be 1 sentence, not 2.

The description should start with the word "Run".

Consider this:

> Run Ethereum DApps with local Ethereum blockchain debugging and deployment